docs/guides/run-jobs-batch.mdx

```diff
@@ -50,4 +50,29 @@ The following example shows how you can partition a long list of circuits into m
 
 <Admonition type="caution">
     If you set `backend=backend` in a V2 primitive, the program is run in job mode, even if it's inside a batch or session context. Setting `backend=backend` is deprecated as of IBM Qiskit Runtime 0.24.0.  Instead, use the `mode` parameter.
+</Admonition>
+
+<span id="batch-details"></span>
+## Determine batch details
+
+For a comprehensive overview of a batch's configuration and status, use the  `batch.details() method`.
```

Comment:
Since the code examples don't show the results anymore, it'd be nice to give a hint on what to use this for. 

```suggestion
For a comprehensive overview of a batch's configuration and status, including its interactive and max TTL, use the  `batch.details() method`.
```